cmd/ctr/snapshot.go

```diff
@@ -231,3 +261,38 @@ var prepareSnapshotCommand = cli.Command{
 		return nil
 	},
 }
+
+type snapshotTreeNode struct {
+	Name     string
+	Parent   string
+	Children []string
+}
+
+func getOrCreateTreeNode(name string, tree map[string]*snapshotTreeNode) *snapshotTreeNode {
+	if node, ok := tree[name]; ok {
+		return node
+	}
+	node := &snapshotTreeNode{
+		Name: name,
+	}
+	tree[name] = node
+	return node
+}
+
+func printTree(tree map[string]*snapshotTreeNode) {
+	for _, node := range tree {
+		// Print for root(parent-less) nodes only
+		if node.Parent == "" {
+			printNode(node.Name, tree, 0)
+		}
+	}
+}
+
+func printNode(name string, tree map[string]*snapshotTreeNode, level int) {
+	node := tree[name]
+	fmt.Printf("%s\\_ %s\n", strings.Repeat("\t", level), node.Name)
```

Comment:
I think the `\t` is too much here, how about just double space so it lines up with `\_`